### DISCUSSION TITLE
Test before plugging into EcoDan

### DISCUSSION THREAD

**Discussion by razorclam** [gekkekoe/esphome-ecodan-hp#80] [General]
Hi,
So far I have flashed the AtomS3 Lite and everything seemed to work OK.
Is there a way to make sure its ready to connect to the EcoDan? 
I have it plugged into my server via USBC but on the ESPHome page it says it is offline.

![2024-11-29 06_57_01-NVIDIA GeForce Overlay DT](https://github.com/user-attachments/assets/db870fb2-ad28-42a3-960a-b978b2ad9367)

I also can not see it on my wifi network

**Comment by gekkekoe** [comment]
if you did it correctly, it should be online in your wifi and you can access it via http. it will complain there that it cannot connect to the heatpump. so probably you did something wrong

**Reply by razorclam** [reply]
Thanks for the replay.
I just unplugged it and used a power only USBC cable and it fired straight up.
Now just got to connect it to the Ecodan

**Reply by gekkekoe** [reply]
don't connect it via USB when running inside heatpump. only grove-cn205 cable!
